css/wee.module.less

```diff
@@ -16,9 +16,12 @@
 .wee-{{moduleName}}-responsive () when (@responsiveEnabled = true) {
 	.check () when not (@mobileLandscapeWidth = false) {
 		@media (min-width: (@mobileLandscapeWidth)) {
-			html {
-				font-family: '"2"';
+			.check () when ({{standalone}} = true) {
```

Comment:
In the rare instance it is standalone it would need to be added, yes. However not everything is represented in the generated module template. I didn't add this since it is such a fringe in my opinion and would usually just be clutter. Can see the argument either way though.